src/main/java/graphql/schema/GraphQLEnumType.java

```diff
@@ -284,6 +306,12 @@ public Builder values(List<GraphQLEnumValueDefinition> valueDefinitions) {
             return this;
         }
 
+        public Builder replaceValues(List<GraphQLEnumValueDefinition> valueDefinitions) {
+            this.values.clear();
+            valueDefinitions.forEach(this::value);
+            return this;
+        }
+
```

Comment:
consistency - replaceValues used here.  I think it should be replaceXXX everywhere??

Reply:
done